lib/web/fetch/request.js

```diff
@@ -1057,6 +1060,12 @@ webidl.converters.RequestInit = webidl.dictionaryConverter([
     key: 'integrity',
     converter: webidl.converters.DOMString
   },
+  {
+    key: 'initiator',
+    converter: webidl.converters.DOMString,
+    // https://fetch.spec.whatwg.org/#concept-request-initiator
+    allowedValues: requestInitiator
+  },
```

Comment:
This is wrong. A Request's request (internal state) contains an initiator, not [RequestInit](https://fetch.spec.whatwg.org/#requestinit)